frontend/src/Head/Head.tsx

```diff
@@ -2,9 +2,14 @@ import style from "./Head.module.scss";
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Button from 'react-bootstrap/Button';
+import { useSelector } from "react-redux";
+import { RootState } from "../store";
 
 
 export default function Head() {
+
+  const user = useSelector((state: RootState) => state.user.user);
```

Comment:
if use useAppSelector from the example you don't need to provide state type each time